aries-site/src/components/home/Highlights.js

```diff
@@ -107,7 +107,9 @@ export const Highlights = () => {
       </Box>
       <HighlightsLayout />
       <Box fill="horizontal" align="center" justify="center" pad="medium">
-        <Button primary label="Show Me More" />
+        <Link href="/showmore" passHref>
```

Comment:
The behavior of this button feels awkward to me. I think we should adjust the top section of the "Show More" page to say something like "All Resources" or "All Pages" or something like that (because currently it feels like I'm being routed to a new home page.

Reply:
As opposed to keeping the original language from the current homepage.

Reply:
@taysea We can change the name of this button. Chris and I had agreed to "Show Me More". After this button has been selected, the user will be directed to our current home page, which the header will be changed to "highlights" to show an overview of all the cards.